modules/serverless-logic-supporting-services-advanced-configuration.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-managing-supporting-services
+
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-supporting-services-advanced-configuration_{context}"]
+= Advanced supporting services configurations
+
+In scenarios where you must apply advanced configurations for supporting services, use the `podTemplate` field in the SonataFlowPlatform Custom Resource (CR). This field allows you to customize the service pod deployment by specifying configurations like the number of replicas, environment variables, container images, and initialization options.
+
+You can configure advanced settings for the service by using the following example:
+
+.Advanced configurations example for the Data Index service
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform-example
+  namespace: example-namespace
+spec:
+  services:
+    dataIndex:
```

Comment:
Here, it can be `jobService` or `dataIndex`. Can we clarify that somehow? Maybe adding another example? That's the only difference.

Reply:
I have added a note at the bottom and added the comment in the yaml snippet as well.